incubator/zookeeper/templates/statefulset.yaml

```diff
@@ -94,9 +94,12 @@ spec:
         volumeMounts:
         - name: datadir
           mountPath: /var/lib/zookeeper
+          subPath: data
```

Comment:
How is the `subPath` related to your change? Does that make sense?

Reply:
When AWS mounts EBS as a PVC, it has some "stuff" already in there, and ZK loves to die saying that the directory is already in use.  Put in a subpath, and it'll be nice and clean, and no more complaints.